.evergreen/config.yml

```diff
@@ -482,7 +488,49 @@ buildvariants:
 - matrix_name: "atlas-connect"
   matrix_spec:
     os-fully-featured: "*"
-    swift-version: "*"
+    swift-version:
+      - "5.1"
+      - "5.2"
+  display_name: "Atlas Connectivity ${swift-version} ${os-fully-featured}"
+  tasks:
+    - ".atlas-connect"
+
+
+# define separate matrices for swift 5.3 that excludes macOS 10.14. unfortunately
+# as of now we cannot remove entire buildvariants via rule from the matrices above
+# so we need to split this up due to inability to test 5.3 on macOS 10.14.
+# see EVG-13092
+- matrix_name: "tests-all"
```

Comment:
does this matrix having the same name as the other cause any issues? Or is it okay to share names?

Reply:
as far as I can tell it's fine - it doesn't seem like matrix name actually gets used for anything besides being prepended at the start of each buildvariant name, e.g. `tests-all__os-fully-featured~macos-1014_swift-version~5.2_ssl-auth~nossl-noauth`.
I think it would only be an issue if duplicate variants were generated by the two matrices since then their names would be identical.